maintainers/maintainer-list.nix

```diff
@@ -16971,6 +16971,12 @@
     githubId = 2141853;
     name = "Bang Lee";
   };
+  qwqawawow = {
+    email = "eihqnh@outlook.com";
+    github = "qwqawawow";
+    githubId = 40905037;
+    name = "qwqawawow";
+  };
```

Comment:
please split of this change into a separate commit titled `maintainers: add qwqawawow`

Reply:
Corrected.Thank you very much. I didn't notice the guide....

Reply:
Corrected.Thank you very much. I didn't notice the guide....